frontend/js/apiKeys/teamCard.tsx

```diff
@@ -51,10 +52,17 @@ function TeamCard(props: Props) {
   return (
     <Panel border className="apikeyCard">
       <Undertittel>{team}</Undertittel>
-      <Element style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+      <div className="keyWrapper">
         <KeyIcon />
-        {key}
-      </Element>
+        <Element className="key">{hidden ? '*'.repeat(key.length) : key}</Element>
```

Comment:
I'll answer my own question: No
```suggestion
        <Element className="key">{hidden ? '*'.repeat(16) : key}</Element>
```

I suggest this to fix it =)

Reply:
Is this because of security? In my opinion I think it works better when it is the same length as the secret because the show/hide button doesn't move around when you click it. I would personally be very mildly annoyed if it moved around 😄

Reply:
> Is this because of security?

Kind of, yes - but also:
> (...) because the show/hide button doesn't move around when you click it. (...)

What _typical_ (read: frequently applied/used/done) use-case requires you to toggle these back and forth?

I struggle to see use-cases (which involve this button that makes sense a user would frequently need/use/require) besides:

1. Copy-paste button doesn't work, so un-hide to manually copy-paste
1. *or* you've got an APIKEY on clipboard, and want to "un-hide" them all to see which one matches in browser

Reply:
A fix could be to set it to (say) `64 * '*'` =)

Reply:
All hashes are always the same length; this would not improve security.